lib/configure.js

```diff
@@ -479,19 +479,32 @@ PythonFinder.prototype = {
   },
 
   failNoPython: function failNoPython () {
-    var errmsg =
-        'Can\'t find Python executable "' + this.python +
-        '", you can set the PYTHON env variable.'
-    this.callback(new Error(errmsg))
+    const err = new Error(
+      '\n******************************************************************\n' +
+      `node-gyp can't use "${this.python}",\n` +
```

Comment:
Did you mean to change the wording from "can't find" to "can't use"?

Reply:
Didn't notice this PR before doing my own fix. Solves the above issue and detects if there is a wrong version + a possible infinite loop fix. See  https://github.com/nodejs/node-gyp/pull/1325